### DISCUSSION TITLE
The train problem

### DISCUSSION THREAD

**Discussion by hero-y** [STVIR/pysot#212] [General]
Hello,I want to use the siamrpn_r50_l234_dwxcorr_8gpu for train,and I only use the COCO,
![image](https://user-images.githubusercontent.com/45655880/67174176-b8877b80-f3f3-11e9-9317-c0b77f984af1.png)
Then I use the commend like 
python -m torch.distributed.launch --nproc_per_node=1  --master_port=2334 tools/train.py --cfg experiments/siamrpn_r50_l234_dwxcorr_8gpu/config.yaml
but it shows subprocess.CalledProcessError,returned non-zero exit status 1 and ValueError: Type mismatch (<class 'str'> vs. <class 'NoneType'>) with values ( vs. None) for config key: BACKBONE.PRETRAINED
![image](https://user-images.githubusercontent.com/45655880/67174269-0a300600-f3f4-11e9-9a42-8f39d65143d2.png)
Can you help me ,Thanks!

**Comment by adrift00** [comment]
Maybe you haven't download the pretained backbone model.

**Comment by ZhiyuanChen** [comment]
It seems your config file was not properly configured.

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing it for now.
Please reopen it or create a new issue if you have further question.
